pytest.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:jessie
+
+ENV GITDIR /etc/.pihole
+ENV SCRIPTDIR /opt/pihole
+
+RUN mkdir -p $GITDIR $SCRIPTDIR /etc/pihole
+ADD . $GITDIR
```

Comment:
pretty inefficient / non functional Dockerfile image that just dumps all files into it for executing through bash...through pytest, to assert return codes and output.